docs/dev/development.adoc

```diff
@@ -66,9 +66,9 @@ Read the Makefile itself for more information.
 === PR review process
 
 . Once you submit a PR, the @openshift-ci-robot automatically requests two reviews from reviewers and suggests an approver based on `OWNERS` files.
-. After a reviewer is satisfied with the changes he adds `/lgtm` (looks good to me) as a comment to the PR. This applies the *lgtm* label.
-. The approver then reviews the PR and if satisfied, adds
-`/approve` as a comment to the PR. This applies the *approve* label.
+. Each PR requires *one* code review (lgtm label) and *one* approve (approved label).
+. After a code reviewer is satisfied with the changes he adds `/lgtm` (looks good to me) as a comment to the PR. Which applies the *lgtm* label.
+. The approver then reviews the PR and if satisfied, adds `/approve` as a comment to the PR. Which applies the *approve* label.
```

Comment:
We should remove `then` as it seems to indicate that `lgtm` needs to happen first

Reply:
Thanks :)